src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -307,16 +347,46 @@ void call(final ClientRequestRound round, final int tryId) {
     final TableConfiguration tableConfig = tableConfig_.get();
     final ReplicaConfiguration handle =
         tableConfig.replicas.get(round.getOperator().get_gpid().get_pidx());
-    if (handle.session != null) {
-      handle.session.asyncSend(
+
+    // send request to primary session
+    if (handle.primarySession != null) {
+      // if it's not write operation and backup request is enabled, schedule to send to secondaries
+      if (!round.operator.isWrite && isEnableBackupRequest()) {
+        round.backupRequstTask =
+            executor_.schedule(
+                new Runnable() {
+                  @Override
+                  public void run() {
+                    for (ReplicaSession session : handle.secondarySessions) {
+                      session.asyncSend(
+                          round.getOperator(),
+                          new Runnable() {
```

Comment:
Prefer to use lambda in this place in order to reduce embed levels.

Reply:
At very beginning, I used lambda. But after format, these code is like this:
```
              handle.secondarySessions.forEach(
                  session ->
                      session.asyncSend(
                          round.getOperator(),
                          new Runnable() {
                            @Override
                            public void run() {
                              onRpcReply(round, tryId, tableConfig.updateVersion, session.name());
                            }
                          },
                          round.timeoutMs,
                          true));
```
I think use for loop is more user-friendly:
```
                    for (ReplicaSession session : handle.secondarySessions) {
                      session.asyncSend(
                          round.getOperator(),
                          new Runnable() {
                            @Override
                            public void run() {
                              onRpcReply(round, tryId, tableConfig.updateVersion, session.name());
                            }
                          },
                          round.timeoutMs,
                          true);
                    }
```

Reply:
`ReplicaSession session : handle.secondarySessions` ->`ReplicaSession secondarySession : handle.secondarySessions`